src/pages/ProductListPage/FilterSection/FilterProduct.jsx

```diff
@@ -1,82 +1,263 @@
+                    <div>
+                        <input type="radio" id="4stars" name="drone" value="4"
+                            onClick={(e)=>setRating(e.target.value)}/>
+                        <label htmlFor="4stars">4 stars and above</label>
+                    </div>
+                    
+                    <div>
+                        <input type="radio" id="3stars" name="drone" value="3"
+                        onClick={(e)=>setRating(e.target.value)}/>
+                        <label htmlFor="3stars">3 stars and above</label>
+                    </div>
+                    
+                    <div>
+                        <input type="radio" id="2stars" name="drone" value="2"
+                        onClick={(e)=>setRating(e.target.value)}/>
+                        <label htmlFor="2stars">2 stars and above</label>
+                    </div>
+                    <div>
+                        <input type="radio" id="1stars" name="drone" value="1"
+                        onClick={(e)=>setRating(e.target.value)}/>
+                        <label htmlFor="1stars">1 stars and above</label>
+                    </div>
 
-            </div>
-            <div className="sort-by-conatianer">
-                <h4>Sort by</h4>
-                <div>
-                    <input type="radio" id="lth" name="price" value="lth"
-                        checked />
-                    <label htmlFor="lth">Price - Low to High </label>
                 </div>
-                
-                <div>
-                    <input type="radio" id="htl" name="price" value="htl" />
-                    <label htmlFor="htl">Price - High to Low</label>
+                {/*  feature : "waterResistant NoLace GhostLace Lace" */}
+                <div className="product-page-category">
+                    <h4>Feature </h4>
+                    <div>
+                        <input className="checkbox-sneakers" type="checkbox" name="waterResistant" value="waterResistant"
+                        onClick={()=>{
+                            setFeature(()=>( feature.includes("waterResistant") 
+                                                    ? [...feature.filter(item=>item !=="waterResistant")]
+                                                    : [...feature,"waterResistant"])
+                                                    )}}/>
+                        <label htmlFor="waterResistant">Water Resistant</label>
+                    </div>
+                    <div>
+                        <input className="checkbox-vans" type="checkbox" name="NoLace" value="NoLace"
+                        onClick={()=>{
+                            setFeature(()=>( feature.includes("NoLace") 
+                                                    ? [...feature.filter(item=>item !=="NoLace")]
+                                                    : [...feature,"NoLace"])
+                                                    )}}/>
```

Comment:
You can write only one function and can handle all the category by passing arguments.

Reply:
okay done